srcpkgs/OpenJK/template

```diff
@@ -0,0 +1,47 @@
+# Template file for 'OpenJK'
+pkgname=OpenJK
+version=0.0.20210922
+revision=1
+archs="i686 x86_64"
+_gitrev=43e9a3da555a487e23081bbd03101e7c926a1e83
+wrksrc="${pkgname}-${_gitrev}"
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/opt/${pkgname} -DBuildJK2SPEngine=on -DBuildJK2SPGame=on -DBuildJK2SPRdVanilla=on"
+makedepends="libjpeg-turbo-devel libpng-devel zlib-devel SDL2-devel"
+depends="SDL2"
+short_desc="Open Source Implementation of Jedi Knight II and III Engine"
+maintainer="Kenneth Dodrill <kmdodrill@protonmail.com>"
+license="GPL-2.0-only"
+homepage="https://github.com/JACoders/OpenJK"
+distfiles="https://github.com/JACoders/OpenJK/archive/${_gitrev}.tar.gz"
+checksum=efd2e3d96fe449819c4276dfea7ae7d591591ee0e651167844110e073f2edb3c
+
+post_install() {
+	vmkdir usr/bin
+	for _bin in 'openjk' 'openjk_sp' 'openjkded'; do
+		touch "${DESTDIR}/usr/bin/${_bin}"
+		cat > "${DESTDIR}/usr/bin/${_bin}" << EOF
+#! /bin/bash
+cd /opt/${pkgname}/JediAcademy
+exec ./${_bin}.${XBPS_MACHINE} \s@
```

Comment:
this should not use XBPS_MACHINE, instead it should ask for `uname -m` when the script is run